collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -282,7 +283,7 @@ public ReferenceLayerSettingsNavigator providesReferenceLayerSettingsNavigator()
                         @Provides
                         @Override
                         public MapFragmentFactory providesMapFragmentFactory() {
-                            return applicationComponent.mapProvider();
+                            return applicationComponent.mapFragmentFactory();
```

Comment:
I think we should really be using the `MapFragmentFactory` interface wherever this in injected. `MapProvider` is just the implementation of that interface.